api/lib/request/mockRequest.ts

```diff
@@ -1,5 +1,4 @@
 const request = require('request-promise')
```

Comment:
why doesnt this need comma? if others require it

Reply:
will discuss over slack, essentially the person that wrote this was a JS heretic.